--- a/chrome/browser/chrome_content_browser_client.cc
+++ b/chrome/browser/chrome_content_browser_client.cc
@@ -232,6 +232,8 @@
 #include "components/autofill/core/common/autofill_features.h"
 #include "components/autofill/core/common/autofill_switches.h"
 #include "components/blocked_content/popup_blocker.h"
+#include "components/boring/adblock/adblock_throttle.h"
+#include "components/boring/core/boring_prefs.h"
 #include "components/captive_portal/core/buildflags.h"
 #include "components/content_settings/browser/page_specific_content_settings.h"
 #include "components/content_settings/browser/ui/javascript_optimizer_setting.h"
@@ -6067,6 +6069,12 @@
     result.push_back(std::move(signin_throttle));
   }
 
+  auto boring_adblock_throttle =
+      boring::AdblockThrottle::MaybeCreate(request, browser_context);
+  if (boring_adblock_throttle) {
+    result.push_back(std::move(boring_adblock_throttle));
+  }
+
   return result;
 }
 
