--- a/chrome/browser/chrome_content_browser_client_receiver_bindings.cc
+++ b/chrome/browser/chrome_content_browser_client_receiver_bindings.cc
@@ -27,6 +27,8 @@
 #include "chrome/common/buildflags.h"
 #include "chrome/common/chrome_features.h"
 #include "components/autofill/content/browser/content_autofill_driver_factory.h"
+#include "components/boring/adblock/adblock_checker_impl.h"
+#include "components/boring/adblock/mojom/adblock.mojom.h"
 #include "components/content_capture/browser/onscreen_content_provider.h"
 #include "components/metrics/call_stacks/call_stack_profile_collector.h"
 #include "components/offline_pages/buildflags/buildflags.h"
@@ -214,6 +216,9 @@
   registry->AddInterface<metrics::mojom::CallStackProfileCollector>(
       base::BindRepeating(&metrics::CallStackProfileCollector::Create));
 
+  registry->AddInterface<boring::mojom::AdblockChecker>(
+      base::BindRepeating(&boring::AdblockCheckerImpl::Bind));
+
   if (NetBenchmarking::CheckBenchmarkingEnabled()) {
     Profile* profile =
         Profile::FromBrowserContext(render_process_host->GetBrowserContext());
