--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -713,6 +713,9 @@
     "//components/blocked_content",
     "//components/bookmarks/browser",
     "//components/bookmarks/managed",
+    "//components/boring/core",
+    "//components/boring/lists",
+    "//components/boring/serp",
     "//components/breadcrumbs/core",
     "//components/breadcrumbs/core:status",
     "//components/browser_apis/tab_strip/types",
